advanced/i2c-sensor-reading/.cargo/config.toml

```diff
@@ -1,36 +1,20 @@
 [build]
-# Uncomment the relevant target for your chip here (ESP32, ESP32-S2, ESP32-S3 or ESP32-C3)
-#target = "xtensa-esp32-espidf"
-#target = "xtensa-esp32s2-espidf"
-#target = "xtensa-esp32s3-espidf"
 target = "riscv32imc-esp-espidf"
 
-[target.xtensa-esp32-espidf]
```

Comment:
> If the plan is to possibly make this training usable with our other chips, does it make sense to remove these now?

I guess it depends on the approach that we take for adding new targets. If we plan to use features/cfgs it does not make sense, but if we go for a branch per target approach it may make sense.

I am happy to wait until we decide which approach to take, and add it back if needed.

Reply:
Ah yeah, you're right, until we know what path we're going to go down we should clean this up as much as possible.

Reply:
Great, PR should be ready for review then. As you mentioned, let's address this once we know what path we're going!